### PR TITLE
5.10: Fix warnings

### PR DESCRIPTION
**Pull request by MrCry0**
The main change: create a separate config for disabling non-x86 options previously enabled by including common-pc configs.

Fixed configs for:

- imx7ulpea-ucom
- apalis-imx6
- apalis-imx8
- imx6ulevk
- imx6ullevk

### CODE REVIEW

**Review by angolini — APPROVED** [review]
LGTM

File: bsp/imx/apalis-imx8.scc [hunk 1]
```diff
@@ -46,4 +46,7 @@ include features/tee/tee.scc
 
 include cfg/usb-mass-storage.scc
 
+# Exclude x86/ACPI features
+include cfg/non-x86.cfg
```
Comment (ricardosalveti) [inline bsp/imx/apalis-imx8.scc:50]: While I would prefer fixing the fragments the other way around (moving stuff that is specific to x86 to the respective x86 boards/bsps), this is better than what we have now and easier to maintain, so fine with it.

**Review by ricardosalveti — APPROVED** [review]
LGTM

File: bsp/imx/apalis-imx8.scc (hunk 1 above)
Comment (MrCry0) [reply bsp/imx/apalis-imx8.scc:50]: Me either :)  We could do it a bit later when having some free time :)

**Pull request merged**